src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/SyntaxKindEx.cs

```diff
@@ -14,20 +14,22 @@ internal static class SyntaxKindEx
     {
         // The code style layer does not currently need access to any syntax defined in newer versions of Roslyn. This
         // type is included as an example should this change in future updates.
-#if false
-        public const SyntaxKind DotDotToken = (SyntaxKind)8222;
+        public const SyntaxKind FunctionPointerType = (SyntaxKind)9056;
+
+#if !CODE_STYLE
```

Comment:
Note had to switch the condition here; I don't think the other way could have worked, since the code style layer can't see both copies.

Reply:
(This was independently fixed in master by somebody else.)